_studio/mfx_lib/cmrt_cross_platform/include/cmvm.h

```diff
@@ -967,7 +967,6 @@ matrix<T, R, C>::matrix(const matrix<T2, R2, C2>& src, const uint sat)
     : array_2d_(data)
 #endif
 {
-    static const bool conformable = check_true<R*C == R2*C2>::value;
```

Comment:
We talked with Oleg today and he noted this, but had no time to post yet. This wonderful code is actually a static_error:)! And should be used as such: CM_STATIC_ERROR(R*C == R2*C2, "matrices have different dimensions");